setup.py

```diff
@@ -111,6 +111,22 @@ def get_libfuzzer_lib():
   return libfuzzer
 
 
+def get_libafl_lib():
+  """Returns path to the LibAFL .a library."""
+  libafl_lib = os.getenv("LIBAFL_LIB", "")
+  if libafl_lib:
+    return libafl_lib
+  current_path = os.path.dirname(os.path.realpath(__file__))
+  try:
+    libafl = subprocess.check_output(
+        [current_path + "/setup_utils/find_libafl.sh"])
```

Comment:
nit: I don't think I see this file in the directory-- am I missing something, or will a subsequent commit add this?